### ISSUE TITLE
Searching is not consistent in Default Query

### ISSUE THREAD

**Issue by adulbrich** [BlueBrain/nexus#1098]
1) Load following list https://bbp.epfl.ch/nexus/web/admin/bbp/agents?shareList=JTdCJTIybmFtZSUyMiUzQSUyMlNhbmRyaW5lJTIyJTJDJTIydmlldyUyMiUzQSUyMm54diUzQWRlZmF1bHRFbGFzdGljU2VhcmNoSW5kZXglMjIlMkMlMjJpZCUyMiUzQSUyMjcwMmRhYWM3LWFkMWItNDQyNS04OWViLWU2ZWI1NWZmMDg4MSUyMiUyQyUyMnF1ZXJ5JTIyJTNBJTdCJTIyc2l6ZSUyMiUzQTEwMCUyQyUyMmRlcHJlY2F0ZWQlMjIlM0FmYWxzZSUyQyUyMnElMjIlM0ElMjJTYW5kcmluZSUyMiU3RCU3RA==
2) Search for sandrine, nothing happens
3) Search for maria, results found
reported by @annakristinkaufmann

**Comment by bogdanromanx** [comment]
@adulbrich we're unable to confirm this problem, the provided URL ends in page not found. Could you please let us know if the issue still occurs?

**Comment by adulbrich** [comment]
@bogdanromanx I updated the link (sorry about that), the problem is still present.
Resource 6fd6f0e5-80bb-4da4-add3-86bc67ef29a3 has a givenName Sandrine but the search does not yield any result while for Maria it works.

**Comment by umbreak** [comment]
The problem is that when the `q` parameter gets handled in nexus, we lowercase it (which is useful for UUIDs). However our custom analyzer does not lowercase keys. We have to [add the lowercase filter](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lowercase-tokenfilter.html#analysis-lowercase-tokenfilter-analyzer-ex) to our default elasticsearch mappings in order to support it.

As a note for myself when building 1.5.x. We will need a default index which looks like (to be reviewed):
```json
{
  "settings": {
    "analysis": {
      "analyzer": {
        "nexus": {
          "type": "custom",
          "tokenizer": "classic",
          "filter": [
            "my_multiplexer"
          ]
        }
      },
      "filter": {
        "my_multiplexer": {
          "type": "multiplexer",
          "filters": [
            "uuid_type_4",
            "word_delimiter_nexus",
            "lowercase"
          ]
        },
        "uuid_type_4": {
          "type": "pattern_capture",
          "preserve_original": true,
          "patterns": [
            "([a-f0-9]{8})\\-([a-f0-9]{4})\\-([a-f0-9]{4})\\-([a-f0-9]{4})\\-([a-f0-9]{12})"
          ]
        },
        "word_delimiter_nexus": {
          "type": "word_delimiter",
          "split_on_numerics": false
        }
      }
    }
  },
  "mappings": {
    "properties": {
      "@type": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "@id": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "prefLabel": {
        "type": "text",
        "fields": {
          "raw": {
            "type": "keyword"
          }
        },
        "copy_to": "_all_fields"
      },
      "label": {
        "type": "text",
        "fields": {
          "raw": {
            "type": "keyword"
          }
        },
        "copy_to": "_all_fields"
      },
      "name": {
        "type": "text",
        "fields": {
          "raw": {
            "type": "keyword"
          }
        },
        "copy_to": "_all_fields"
      },
      "_rev": {
        "type": "long",
        "copy_to": "_all_fields"
      },
      "_deprecated": {
        "type": "boolean",
        "copy_to": "_all_fields"
      },
      "_createdAt": {
        "type": "date",
        "copy_to": "_all_fields"
      },
      "_updatedAt": {
        "type": "date",
        "copy_to": "_all_fields"
      },
      "_createdBy": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_updatedBy": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_constrainedBy": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_project": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_self": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_incoming": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_outgoing": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_original_source": {
        "type": "text",
        "analyzer": "nexus",
        "copy_to": "_all_fields"
      },
      "_bytes": {
        "type": "long",
        "copy_to": "_all_fields"
      },
      "_mediaType": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_location": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_filename": {
        "type": "keyword",
        "copy_to": "_all_fields"
      },
      "_digest": {
        "type": "nested",
        "properties": {
          "_algorithm": {
            "type": "keyword",
            "copy_to": "_all_fields"
          },
          "_value": {
            "type": "keyword",
            "copy_to": "_all_fields"
          }
        }
      },
      "_storage": {
        "type": "nested",
        "properties": {
          "_rev": {
            "type": "long",
            "copy_to": "_all_fields"
          },
          "@id": {
            "type": "keyword",
            "copy_to": "_all_fields"
          }
        }
      },
      "_all_fields": {
        "type": "text",
        "analyzer": "nexus"
      }
    },
    "dynamic": false
  }
}
```

**Comment by bogdanromanx** [comment]
Fixed in #2051
